src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+
+body, html, section {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 20px;
+  height: 20px;
+  background-image: url('./images/star.svg');
+  background-size: cover;
+  margin-right: 5px;
+}
+
+.stars--0 .stars__star:nth-child(-n+0),
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
```

Comment:
There is a logical error in the ':nth-child' selectors. The '-n+0' selector in '.stars--0 .stars__star:nth-child(-n+0)' will not select any elements, as it effectively means 'select none'. Similarly, the other selectors should be adjusted to correctly select the intended number of active stars. For example, '.stars--1 .stars__star:nth-child(-n+1)' should be '.stars--1 .stars__star:nth-child(-n+2)' to select the first star, and so on.